File: src/Entity/Experience.php
<?php

namespace App\Entity;

use App\Repository\ExperienceRepository;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;

/**
 * @ORM\Entity(repositoryClass=ExperienceRepository::class)
 */
class Experience
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=255)
     * * @Assert\Length(
     *      min = 5,
     *      max = 50,
     *      minMessage = "Your Title must be at least {{ limit }} characters long",
     *      maxMessage = "Your  Title cannot be longer than {{ limit }} characters"
     * )
     */
    private $Titre;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $Description;

    /**
     * @ORM\Column(type="integer", length=1)
     *
     */
    private $Note;

    /**
     * @ORM\Column(type="integer", length=11)
     */
    private $Points;

    /**
     * @ORM\Column(type="integer", length=11)
     */
    private $Id_client;

    /**
     * @ORM\Column(type="datetime")
     */
    private $Date;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\File(mimeTypes={"image/jpeg"})
     */
    private $Image;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $endroit;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getTitre(): ?string
    {
        return $this->Titre;
    }

    public function setTitre(string $Titre): self
    {
        $this->Titre = $Titre;

        return $this;
    }

    public function getDescription(): ?string
    {
        return $this->Description;
    }

    public function setDescription(string $Description): self
    {
        $this->Description = $Description;

        return $this;
    }

    public function getNote(): ?int
    {
        return $this->Note;
    }

    public function setNote(int $Note): self
    {
        $this->Note = $Note;

        return $this;
    }

    public function getPoints(): ?string
    {
        return $this->Points;
    }

    public function setPoints(string $Points): self
    {
        $this->Points = $Points;

        return $this;
    }

    public function getIdClient(): ?string
    {
        return $this->Id_client;
    }

    public function setIdClient(string $Id_client): self
    {
        $this->Id_client = $Id_client;

        return $this;
    }

    public function getDate(): ?\DateTimeInterface
    {
        return $this->Date;
    }

    public function setDate(\DateTimeInterface $Date): self
    {
        $this->Date = $Date;

        return $this;
    }

    public function getImage()
    {
        return $this->Image;
    }

    public function setImage( $Image)
    {
        $this->Image = $Image;

        return $this;
    }

    public function getEndroit(): ?string
    {
        return $this->endroit;
    }

    public function setEndroit(string $endroit): self
    {
        $this->endroit = $endroit;

        return $this;
    }
}

File: JAVA3/src/entité/Experience.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entité;


import java.time.LocalDateTime;
import java.util.Objects;
import javafx.beans.property.SimpleIntegerProperty;
import javafx.beans.property.SimpleStringProperty;

/**
 *
 * @author Administrateur
 */
public class Experience {
    private SimpleIntegerProperty id; 
    private SimpleStringProperty Titre;
    private SimpleStringProperty Description;
    private SimpleIntegerProperty Note;
    private SimpleIntegerProperty points;
    private SimpleIntegerProperty id_client;
    private LocalDateTime Date;
    private SimpleStringProperty endroit;
    private String image;

    public Experience(String Titre,String  Description, int Note, int id_client, String endroit,String image) {
        this.Titre = new SimpleStringProperty(Titre);
        this.Description = new SimpleStringProperty(Description);
        this.Note =new SimpleIntegerProperty (Note);
        this.id_client = new SimpleIntegerProperty (id_client);
        this.endroit = new SimpleStringProperty(endroit);
        this.image=image;
    }
 public Experience(String Titre,String  Description, int Note, int id_client, String endroit) {
        this.Titre = new SimpleStringProperty(Titre);
        this.Description = new SimpleStringProperty(Description);
        this.Note =new SimpleIntegerProperty (Note);
        this.id_client = new SimpleIntegerProperty (id_client);
        this.endroit = new SimpleStringProperty(endroit);
        
    }
    public Experience() {
    }

    @Override
    public int hashCode() {
        int hash = 7;
        hash = 19 * hash + Objects.hashCode(this.id);
        return hash;
    }

    @Override
    public boolean equals(Object obj) {
        if (this == obj) {
            return true;
        }
        if (obj == null) {
            return false;
        }
        if (getClass() != obj.getClass()) {
            return false;
        }
        final Experience other = (Experience) obj;
        if (!Objects.equals(this.id, other.id)) {
            return false;
        }
        return true;
    }

    public int getId() {
        return id.get();
    }

    public void setId(int id) {
       this.id = new SimpleIntegerProperty(id);
    }

    public String getTitre() {
        return Titre.get();
    }
public SimpleStringProperty getTitreProperty(){
        return Titre;
    }
    public SimpleStringProperty getDescriptionProperty(){
        return Description;
    }
    public void setTitre(String Titre) {
        this.Titre =  new SimpleStringProperty(Titre);
    }

    public String getDescription() {
        return Description.get();
    }

    public void setDescription(String Description) {
        this.Description = new SimpleStringProperty(Description);
    }

    public int getNote() {
        return Note.get();
    }

    public void setNote(int Note) {
        this.Note = new SimpleIntegerProperty(Note);
    }

    public int getPoints() {
        return points.get();
    }

    public void setPoints(int points) {
        this.points = new SimpleIntegerProperty(points);
    }

    public int getId_client() {
        return id_client.get();
    }

    public void setId_client(SimpleIntegerProperty id_client) {
        this.id_client = id_client;
    }

    public LocalDateTime getDate() {
        return Date;
    }

    public void setDate(LocalDateTime Date) {
        this.Date = Date;
    }

    public String getEndroit() {
        return endroit.get();
    }

    public void setEndroit(SimpleStringProperty endroit) {
        this.endroit = endroit;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    

    @Override
    public String toString() {
        return  "***" + Titre.get() +"***    :"+ Description.get()  ;
    }

    
    
}
